docs/dictionary/constant/nothing.lcdoc

```diff
@@ -0,0 +1,19 @@
+Name: nothing
+
+Type: constant
+
+Syntax: nothing
+
+Summary:
+The value with no type
```

Comment:
I'm not sure this is an accurate description. `The (unique) value representing the notion of no value`.